package_metrics/metrics.py

```diff
@@ -16,7 +16,7 @@ def build_row(behind, name, current, latest):
         if delta:
             behind = ".".join(str(v) for v in delta)
         else:
-            continue
+            behind = "n/a"
```

Comment:
Flagging this as it was briefly discussed in https://github.com/dimagi/package-metrics/pull/3. Given the full context, what are your thoughts now @millerdev? To be clear, the output would look something like:
```
Behind   Package                      Latest       Version
n/a      At.js                        unknown       1.5.3
```

Reply:
Looks good to me.